dev-python/regex/files/regex-2021.4.4-pypy3-fix-test_empty_array.patch

```diff
@@ -0,0 +1,35 @@
+From: Arthur Zamarin <arthurzam@gmail.com>
```

Comment:
Upstreamable? If so, pop a link in.

Good work, by the way!

Reply:
Well, I posted the PR [here](https://bitbucket.org/mrabarnett/mrab-regex/pull-requests/5), lets see their response...